service/src/main/java/com/epam/esm/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,94 @@
+package com.epam.esm.service.impl;
+
+import com.epam.esm.dao.CertificateDao;
+import com.epam.esm.dao.OrderDao;
+import com.epam.esm.service.exception.CreateEntityInternalException;
+import com.epam.esm.domain.dto.CreateOrderDTO;
+import com.epam.esm.domain.dto.OrderDTO;
+import com.epam.esm.domain.entity.Certificate;
+import com.epam.esm.domain.entity.Order;
+import com.epam.esm.dao.impl.CertificateDaoImpl;
+import com.epam.esm.service.CertificateService;
+import com.epam.esm.service.OrderService;
+import com.epam.esm.service.exception.IdNotExistException;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
+public class OrderServiceImpl implements OrderService {
+
+    OrderDao orderDao;
+    ModelMapper modelMapper;
+    CertificateService certificateService;
+    CertificateDao certificateDao;

```

Comment:
Modifiers missed

Reply:
This class should work with CertificateService, but not with CertificateDao.

Reply:
Thank you for remark. fixed